includes/Connector.class.php

```diff
@@ -184,7 +184,16 @@ public function curl($url, $params_data = array(), $verb = "", $custom_method =
 		}
 		curl_close($request);
 		$debug_str1 .= "curl_close(\$ch);\n";
-		$object = json_decode($response);
+		switch ($this->output) {
+			case 'xml':
+				$object = simplexml_load_string($response);
+				break;
+			case 'json':
+				$object = json_decode($response);
+				break;
+			default:

```

Comment:
Shouldn't the default be the existing (for all users already using this) - json_decode?